addons/im_livechat/data/im_livechat_channel_demo.xml

```diff
@@ -23,118 +23,133 @@
             <field name="livechat_operator_id" ref="base.partner_admin"/>
             <field name="name">Visitor #234, Mitchell Admin</field>
             <field name="anonymous_name">Visitor #234, Mitchell Admin</field>
+            <field name="channel_last_seen_partner_ids" eval="[(0, 0, {'partner_id': ref('base.partner_admin')}), (0, 0, {'partner_id': ref('base.public_partner')})]"/>
```

Comment:
If you don't need to write any specific field of `channel.partner`, it's easier to write on `channel_partner_ids` instead of `channel_last_seen_partner_ids`.

(Same remark for all others)

Reply:
@seb-odoo It is not working. Channels are not showing in discuss sidebar if we use 'channel_partner_ids'.

Reply:
I guess if they don't show it is because they have `is_pinned` to `False`? But `True` should be the default value, I thought we fixed that when defining `_inverse_channel_partner_ids`?

Internally writing on `channel_partner_ids` should be equal to creating `mail.channel.partner` thanks to that method, so it would be nice to know why it doesn't work here?

Reply:
Hello @seb-odoo 

as we have discussed there is an issue with the current partner is added to the channel partner.
so that when data is created it is adding odoobot as the partner
and in the sidebar channel name 'odoobot' appears as the channel name.

Initially, I have tried to remove the odoobot from the channel partners
can I have your opinion on this case?

Thanks